apps/docs/app/features/routes/ressurser/design-tokens/BreakpointTokens.tsx

```diff
@@ -1,6 +1,80 @@
-import { BoxProps } from "@vygruppen/spor-react";
+import {
+  BoxProps,
+  Box,
+  Code,
+  Stack,
+  Table,
+  Tbody,
+  Td,
+  Text,
+  Th,
+  Thead,
+  Tr,
+} from "@vygruppen/spor-react";
 import { SharedTokenLayout } from "./SharedTokenLayout";
+import tokens from "@vygruppen/spor-design-tokens";
+import { useTokenFormatter } from "../../../tokens/useTokenFormatter";
+
+type BreakpointToken = {
+  name: string;
+  key: "sm" | "md" | "lg";
+};
+
+const breakpointTokens: BreakpointToken[] = [
+  {
+    name: "Tablet",
+    key: "sm",
+  },
+  {
+    name: "Desktop",
+    key: "md",
+  },
+  {
+    name: "Widescreen",
+    key: "lg",
+  },
+];
 
 export function BreakpointTokens(props: BoxProps) {
-  return <SharedTokenLayout {...props} title="Breakpoints"></SharedTokenLayout>;
+  return (
+    <SharedTokenLayout {...props} title="Breakpoints">
+      <BreakpointTokensTable />
+    </SharedTokenLayout>
+  );
 }
+
+type BreakpointTokenTableProps = BoxProps;
+
+const BreakpointTokensTable = (props: BreakpointTokenTableProps) => {
+  const tokenFormatter = useTokenFormatter();
+  return (
+    <Box {...props}>
+      <Table variant="simple" colorScheme="grey">
+        <Thead>
+          <Tr>
+            <Th>Navn</Th>
+            <Th>Verdi</Th>
+            <Th>Kode</Th>
+          </Tr>
+        </Thead>
+        <Tbody>
+          {breakpointTokens.map((token) => (
```

Comment:
Her kan du også bruke _nesten_  samme approach som du gjør med z-index. Vi mangler riktignok navnet, men det kan vi ha for seg.

Først definerer vi en mapping mellom nøklene til tokensa, og hva de skal hete:

```tsx
type Breakpoints = keyof typeof tokens.size.breakpoint;
const breakpointDisplayNames: Record<Breakpoints, string> = {
  "sm": "Mobil",
  "md": "Desktop",
  "lg": "Wide screen".
};

Så henter vi tokens ut fra tokens-objektet - og viser bare "key" om det ikke er noen match (om vi har lagt til nye tokens uten å oppdatere dokumentasjonen f.eks.):

```tsx
{Object.entries(tokens.size.breakpoint).map(([key, token]) => (
  <Tr key={key}>
    <Td>{breakpointDisplayNames[key] || key}</Td>
     ...
)}
```

Reply:
hm ok. Hva med 

```tsx
<Td>{breakpointDisplayNames[key as Breakpoints] || key}</Td>
```
?

Reply:
Det funka fint - fiksa det samtidig som jeg fiksa bygget :)